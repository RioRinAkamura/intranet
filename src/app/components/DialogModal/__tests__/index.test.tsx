import * as React from 'react';
import { render, screen } from '@testing-library/react';

import { DialogModal } from '..';
import { createRenderer } from 'react-test-renderer/shallow';

jest.mock('react-i18next', () => ({
  useTranslation: () => {
    return {
      t: str => str,
      i18n: {
        changeLanguage: () => new Promise(() => {}),
      },
    };
  },
}));

const renderer = createRenderer();

describe('<DialogModal  />', () => {
  let utils;
  // it('should match snapshot', () => {
  //   renderer.render(<DialogModal>Test Modal</DialogModal>);
  //   const renderedOutput = renderer.getRenderOutput();
  //   expect(renderedOutput).toMatchSnapshot();
  // });

  it('shoud render dialog modal', () => {
    const { queryByText, getByRole } = render(
      <DialogModal okText="Submit">Test Modal</DialogModal>,
    );
    expect(getByRole('button')).toBeTruthy();
  });
});
