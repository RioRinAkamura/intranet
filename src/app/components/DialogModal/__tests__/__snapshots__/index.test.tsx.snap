// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DialogModal  /> should match snapshot 1`] = `null`;
